imap_processing/tests/test_cli.py

```diff
@@ -316,6 +320,54 @@ def test_hit_l1a(mock_hit_l1a, mock_instrument_dependencies):
     assert mocks["mock_upload"].call_count == 2
 
 
+@pytest.mark.usefixtures("_unset_metakernel_path")
+def test_pre_processing(spice_test_data_path):
+    """Test coverage for ProcessInstrument.pre_processing method()."""
+    kernels_to_furnish = ["naif0012.tls", "imap_sclk_0000.tsc"]
+    dependency_str = (
+        "["
+        '{"type": "science","files": ["imap_hi_l2a_sensor45-de_20100105_v001.cdf"]},'
+        f'{{"type":"spice","files": ["{'", "'.join(kernels_to_furnish)}"]}},'
+        '{"type": "spice","files": ["imap_2010_104_01.repoint.csv"]}'
+        "]"
+    )
```

Comment:
This is pretty confusing to follow with all the interwoven single and double quotes, what about creating the object and calling `json.dumps()` on it?

`json.dumps([{"type": "science", "files": ["imap_hi_l2a_sensor45-de_20100105_v001.cdf"]}, ...])`